### PR TITLE
EN-5231 : fix logger problem

### PR DESCRIPTION
**Pull request by miiu96**
fix problem with logger

### CODE REVIEW

File: api/logs/logSender.go [hunk 1]
```diff
@@ -103,7 +105,8 @@ func (ls *logSender) waitForPatternMessage() error {
 func (ls *logSender) monitorConnection() {
 	for {
 		mt, _, err := ls.conn.ReadMessage()
-		if mt == websocket.CloseMessage {
+		ls.log.Info("message type", "value", mt)
```
Comment (iulianpascalau) [inline api/logs/logSender.go]: Maybe make this logger message of type Trace

File: api/logs/logSender.go [hunk 2]
```diff
@@ -103,7 +105,8 @@ func (ls *logSender) waitForPatternMessage() error {
...
+		if mt == websocket.CloseMessage || mt == disconnectMessage {
 			_ = ls.writer.Close()
```
Comment (iulianpascalau) [inline api/logs/logSender.go]: As stated, I think it is better to close the writer if any error is received. Maybe add a defer func on monitorConnection?

File: api/logs/logSender.go [hunk 3]
```diff
@@ -9,6 +9,8 @@ import (
 	"github.com/gorilla/websocket"
 )
 
+const disconnectMessage = -1
```
Comment (iulianpascalau) [inline api/logs/logSender.go]: instead of this const might check if there is a const in websocket package (like websocket.CloseMessage)

Comment (bogdan-rosianu) [reply api/logs/logSender.go]: the constant you mention is unexported so it can't be used

Comment (iulianpascalau) [reply api/logs/logSender.go]: ok

File: api/logs/logSender.go (hunk 1 above)
Comment (miiu96) [reply api/logs/logSender.go]: Done.

File: api/logs/logSender.go (hunk 2 above)
Comment (miiu96) [reply api/logs/logSender.go]: Done.

**Pull request merged**